src/store/traction/saphyr/requests/getters.js

```diff
@@ -0,0 +1,6 @@
+const getters = {
+    requests: state => state.requests,
+    requestsRequest: (state, getters, rootState) => rootState.api.traction.pacbio.requests,
```

Comment:
could add a small test for requestsRequest

Reply:
I have added a new issue to review how vuex is setup. I think there are a few getters and other things which are not currently tested.